### PR TITLE
Add surface area to neurite functions.

### PR DESCRIPTION
**Pull request by liesbethvanherpe**

### CODE REVIEW

File: neurom/fst/_neuritefunc.py [hunk 1]
```diff
@@ -81,6 +81,14 @@ def n_leaves(neurites, neurite_type=NeuriteType.all):
     return n_sections(neurites, neurite_type=neurite_type, iterator_type=Tree.ileaf)
 
 
+def total_surface_area_per_neurite(neurites, neurite_type=NeuriteType.all):
```
Comment (mgeplf) [inline neurom/fst/_neuritefunc.py]: to make this a feature that's available from the command line, it has to be named in
neurom/fst/__init__.py, which I think would be a good idea

File: neurom/fst/_neuritefunc.py [hunk 2]
```diff
@@ -81,6 +81,14 @@ def n_leaves(neurites, neurite_type=NeuriteType.all):
...
+    '''Surface area in a collection of neurites.
+
+    The area is defined as the sum of the area of the sections.
+    '''
+    return [neurite.area for neurite in iter_neurites(neurites, filt=is_type(neurite_type))]
+
+
```
Comment (lidakanari) [inline neurom/fst/_neuritefunc.py]: Maybe total_area_per_neurite to be consistent with the terminology in the code (section_area etc)?

File: neurom/fst/__init__.py [hunk 3]
```diff
@@ -85,7 +85,8 @@
     'segment_taper_rates': _nrt.segment_taper_rates,
     'segment_radial_distances': _nrt.segment_radial_distances,
     'segment_meander_angles': _nrt.segment_meander_angles,
-    'principal_direction_extents': _nrt.principal_direction_extents
+    'principal_direction_extents': _nrt.principal_direction_extents,
+    'total_surface_area_per_neurite': _nrt.total_surface_area_per_neurite
```
Comment (mgeplf) [inline neurom/fst/__init__.py]: not sure why principal_direction_extents didn't have a trailing comma, but could you add one to the line you added?

File: neurom/fst/_neuritefunc.py (hunk 2 above)
Comment (liesbethvanherpe) [inline neurom/fst/_neuritefunc.py]: I was inspired by the "soma_surface_area". It's the same to me - which one is preferred?

Comment (lidakanari) [inline neurom/fst/_neuritefunc.py]: It's not the same computations between section and soma (I don't mind whichever is used) but it has to be consistent between section_area etc...

Comment (mgeplf) [inline neurom/fst/_neuritefunc.py]: Sadly, we already have inconsistencies.  I think `total_area_per_neurite` makes sense

**Comment by mgeplf** [comment]
Thanks, and thanks for the nice test

**Pull request merged**